### PR TITLE
add unl, ucar, nist, aau harvester

### PR DESCRIPTION
**Pull request by kms6bn**
starting over - reran unl harvester. will work on test next.

### CODE REVIEW

File: scrapi/harvesters/aau.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+Example API call: http://etd.aau.edu.et/oai/request?verb=ListRecords&metadataPrefix=oai_dc
+'''
+from __future__ import unicode_literals
+
+from scrapi.base import OAIHarvester
+
+
+class AauHarvester(OAIHarvester):
+    short_name = 'aau'
```
Comment (fabianvf) [inline scrapi/harvesters/aau.py]: Can you make this name a little longer?

File: scrapi/harvesters/ucar.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+Example API call: http://opensky.ucar.edu/oai2?verb=ListRecords&metadataPrefix=oai_dc
+'''
+from __future__ import unicode_literals
+
+from scrapi.base import OAIHarvester
+
+
+class UcarHarvester(OAIHarvester):
+    short_name = 'ucar'
```
Comment (fabianvf) [inline scrapi/harvesters/ucar.py]: Can you rename this to `opensky` or something like that? Maybe `ucaropensky`

File: scrapi/harvesters/unl.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+Example API call: http://digitalcommons.unl.edu/do/oai/?verb=ListRecords&metadataPrefix=oai_dc
+'''
+from __future__ import unicode_literals
+
+from scrapi.base import OAIHarvester
+
+
+class UnlHarvester(OAIHarvester):
+    short_name = 'unl'
```
Comment (fabianvf) [inline scrapi/harvesters/unl.py]: Can you make this `unl_digitalcommons` or something like that?

**Comment by fabianvf** [comment]
Looks good :+1: Just a couple shortname changes

**Comment by kms6bn** [comment]
pushed renamed versions, this should be good to go! Let me know if I need to change anything else.

**Comment by fabianvf** [comment]
It looks like you didn't actually change the short name of the harvesters (`aau`, etc)

**Comment by kms6bn** [comment]
sorry about that! it should be updated now

**Pull request closed without merging**